nncf/onnx/graph/model_transformer.py

```diff
@@ -336,7 +348,7 @@ def _insert_quantizer_dequantizer(
         )
         model.graph.initializer.extend([onnx_scale_tensor, onnx_zero_point_tensor])
         model.graph.value_info.extend([onnx_scale_value_info, onnx_zero_point_info])
-        insert_index = onnx_graph.get_node_index(input_nodes[0].name)
+        insert_index = get_node_index(model, input_nodes[0].name)
```

Comment:
Could you please briefly explain why we need an insertion index? Is it not possible to insert at the end?